items.csv

```diff
@@ -1,20 +1,1222 @@
+lime_concrete_powder,8
+lime_concrete_powder,8
+lime_concrete_powder,8
+lime_concrete_powder,8
+lime_dye,2
+lime_dye,2
+lime_dye,2,green_dye,1,white_dye,1
+lime_stained_glass,8,glass,8,lime_dye,1
+lime_stained_glass_pane,16,lime_stained_glass,6
+lime_stained_glass_pane,8,glass_pane,8,lime_dye,1
+lime_terracotta,8,terracotta,8,lime_dye,1
+lime_wool,1,wool,1
+lime_wool,1
+lodestone,1,chiseled_stone_bricks,3,netherite_ingot,1
+lodestone,1,chiseled_stone_bricks,8,netherite_ingot,1
 log,1
-netherquartz,1,netherquartz_ore,1
-netherquartz_ore,1,
+loom,1,string,1,spruce_planks,1,warped_planks,2
+loom,1,string,2,planks,2
+magenta_banner,1,magenta_wool,6,stick,1
+magenta_bed,1
+magenta_bed,1
+magenta_bed,1,magenta_wool,3,planks,3
+magenta_carpet,3,magenta_wool,2
+magenta_carpet,8,white_carpet,8,magenta_dye,1
+magenta_concrete_powder,8
+magenta_concrete_powder,8
+magenta_concrete_powder,8
+magenta_concrete_powder,8
+magenta_concrete_powder,8
+magenta_concrete_powder,8
+magenta_concrete_powder,8
+magenta_concrete_powder,8
+magenta_concrete_powder,8
+magenta_dye,1
+magenta_dye,2
+magenta_dye,2
+magenta_dye,2
+magenta_dye,2,lilac,1
+magenta_dye,3
+magenta_dye,3
+magenta_dye,3
+magenta_dye,4
+magenta_dye,4
+magenta_dye,4
+magenta_dye,4
+magenta_stained_glass,8,glass,8,magenta_dye,1
+magenta_stained_glass_pane,16,magenta_stained_glass,6
+magenta_stained_glass_pane,8,glass_pane,8,magenta_dye,1
+magenta_terracotta,8,terracotta,8,magenta_dye,1
+magenta_wool,1,wool,1
+magenta_wool,1
+magma_block,1,magma_cream,4
+magma_block,1,magma_cream,4
+magma_cream,1
+magma_cream,1
+magma_cream,1,blaze_powder,1,slimeball,1
+map,1,paper,8,compass,1
+melon,1,melon_slice,9
+melon,1,melon_slice,4
+melon_seeds,1
+melon_seeds,1,melon_slice,1
+minecart,1,iron_ingot,3
+minecart,1,iron_ingot,5
+minecart_with_chest,1,chest,1,minecart,1
+minecart_with_furnace,1,furnace,1,minecart,1
+minecart_with_hopper,1,hopper,1,minecart,1
+minecart_with_tnt,1,tnt,1,minecart,1
+mojang_banner_pattern,1
+mojang_banner_pattern,1
+moss_carpet,3,moss_block,2
+mossy_cobblestone,1
+mossy_cobblestone,1
+mossy_cobblestone,1,cobblestone,1,vines,1
+mossy_cobblestone_slab,6,mossy_cobblestone,3
+mossy_cobblestone_slab,6,mossy_cobblestone,3
+mossy_cobblestone_stairs,4,mossy_cobblestone,4
+mossy_cobblestone_stairs,4,mossy_cobblestone,6
+mossy_cobblestone_wall,6,mossy_cobblestone,4
+mossy_cobblestone_wall,6,mossy_cobblestone,6
+mossy_stone_brick_slab,6,mossy_stone_bricks,3
+mossy_stone_brick_slab,6,mossy_stone_bricks,3
+mossy_stone_brick_stairs,4,mossy_stone_bricks,4
+mossy_stone_brick_stairs,4,mossy_stone_bricks,6
+mossy_stone_brick_wall,6,mossy_stone_bricks,4
+mossy_stone_brick_wall,6,mossy_stone_bricks,6
+mossy_stone_bricks,1
+mossy_stone_bricks,1
+mossy_stone_bricks,1,stone_bricks,1,vines,1
+mushroom_stew,1
+mushroom_stew,1
+mushroom_stew,1
+mushroom_stew,1,red_mushroom,1,brown_mushroom,1,bowl,1
+nether_brick_fence,6,nether_bricks,2,nether_brick,2
+nether_brick_fence,6,nether_bricks,4,nether_brick,2
```

Comment:
Two different recipes for the nether brick fence?

Reply:
The one with more required items is the correct one. The first set of data I used for recipes somehow left out the top row of the crafting table.